contracts/EllipseMarketMaker.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity 0.4.18;
+
+import './ERC20.sol';
+import './TokenOwnable.sol';
+
+/// @title Ellipse Market Maker contract.
+/// @dev market maker, using ellipse equation.
+/// @author Tal Beja.
+contract EllipseMarketMaker is TokenOwnable {
```

Comment:
No, because it is not explicitly implement all of the interface functions (it using delegate to lib using the fallback function). so it couldn't be deployed as an abstract contract.